src/parser/mage/shared/modules/features/RuneOfPower.js

```diff
@@ -59,31 +75,46 @@ class RuneOfPower extends Analyzer {
     };
   }
 
-
   get roundedSecondsSuggestionThresholds() {
     return {
       actual: this.roundedSecondsPerCast,
       isLessThan: {
         minor: RUNE_DURATION,
         average: RUNE_DURATION - 1,
-        major: RUNE_DURATION - 3,
+        major: RUNE_DURATION - 2,
       },
       style: 'number',
     };
   }
 
   showSuggestion = true;
-
   suggestions(when) {
-    if (this.showSuggestion) {
-      when(this.damageSuggestionThresholds)
-        .addSuggestion((suggest, actual, recommended) => {
-          return suggest(<>Your <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id} /> damage boost is below the expected passive gain from <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />. Either find ways to make better use of the talent, or switch to <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} />.</>)
+    if (!this.hasROP) {
+      when(SUGGEST_ROP[this.selectedCombatant.specId]).isTrue()
+        .addSuggestion((suggest) => {
+          return suggest(
+            <>
+            It is highly recommended to talent into <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id} /> when playing this spec.
+            While it can take some practice to master, when played correctly it outputs substantially more DPS than <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id} /> or <SpellLink id={SPELLS.MIRROR_IMAGE_TALENT.id} />.
+            </>)
             .icon(SPELLS.RUNE_OF_POWER_TALENT.icon)
-            .actual(`${formatPercentage(this.damageIncreasePercent)}% damage increase from Rune of Power`)
-            .recommended(`${formatPercentage(recommended)}% is the passive gain from Incanter's Flow`);
+            .staticImportance(SUGGESTION_IMPORTANCE.REGULAR);
         });
+      return;
+    }
+
+    if(!this.showSuggestion) {
+      return;
     }
+
+    when(this.damageSuggestionThresholds)
```

Comment:
just to make sure im reading this right, the return after the SuggestRoP suggestion is preventing this suggestion from loading if the player doesnt have Rune of Power, correct? Since the module now loads when you dont have RoP.

Reply:
Correct